checkov/terraform/graph_builder/local_graph.py

```diff
@@ -176,6 +139,21 @@ def get_module_vertices_mapping(self) -> None:
         For each vertex, if it's originated in a module import, add to the vertex the index of the
         matching module vertex as 'source_module'
         """
+        if strtobool(os.getenv('CHECKOV_NEW_TF_PARSER', 'False')):
```

Comment:
can you save the value in the init function? like you did with `self.enable_foreach_handling`